english/write-a-hiku/hiku.md

```diff
@@ -0,0 +1,3 @@
+Git, so wonderful
+Beautiful branch structure
+[Discard the hiku](../)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.